pkg/features/antrea_features.go

```diff
@@ -171,6 +172,45 @@ var (
 		LoadBalancerModeDSR:         {Default: false, PreRelease: featuregate.Alpha},
 	}
 
+	AgentGates = sets.New[string](
+		"AntreaIPAM",
+		"AntreaPolicy",
+		"AntreaProxy",
+		"CleanupStaleUDPSvcConntrack",
+		"Egress",
+		"EndpointSlice",
+		"ExternalNode",
+		"FlowExporter",
+		"IPsecCertAuth",
+		"L7NetworkPolicy",
+		"LoadBalancerModeDSR",
+		"Multicast",
+		"Multicluster",
+		"NetworkPolicyStats",
+		"NodePortLocal",
+		"SecondaryNetwork",
+		"ServiceExternalIP",
+		"SupportBundleCollection",
+		"TopologyAwareHints",
+		"Traceflow",
+		"TrafficControl",
+	)
+
+	ControllerGates = sets.New[string](
+		"AntreaIPAM",
+		"AntreaPolicy",
+		"Egress",
+		"IPsecCertAuth",
+		"L7NetworkPolicy",
+		"Multicast",
+		"Multicluster",
+		"NetworkPolicyStats",
+		"NodeIPAM",
+		"ServiceExternalIP",
+		"SupportBundleCollection",
+		"Traceflow",
+	)
```

Comment:
Why using string here? I think we should use the const variables of featuregates.

Reply:
+1
and it's better to add a comment before these two variables to warm developers to update these two Sets.

Reply:
> Why using string here? I think we should use the const variables of featuregates.

fix: https://github.com/antrea-io/antrea/pull/5314/commits/ebd2c8e510abff7d763a94eb95214a4c226a739f

> and it's better to add a comment before these two variables to warm developers to update these two Sets.

fix: https://github.com/antrea-io/antrea/pull/5314/commits/f1658edfe62541b740fa01583d0b1207e0e3f451